### PR TITLE
point listeners to different targets

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- point listeners to different targets
-
-

## security considerations
None

### CODE REVIEW

File: terraform/stacks/main/outputs.tf [hunk 1]
```diff
@@ -612,5 +612,5 @@ output "tcp_lb_listener_ports" {
 }
 
 output "tcp_lb_security_groups" {
-  value = [module.cf.tcp_lb_security_groups.*, module.stack.bosh_security_group]
+  value = [module.cf.tcp_lb_security_groups.*.name, module.stack.bosh_security_group]
```
Comment (soutenniza) [inline terraform/stacks/main/outputs.tf:615]: does the bosh_security_group need to reference to name too?

Comment (bengerman13) [reply terraform/stacks/main/outputs.tf:615]: it's already a name

**Review by soutenniza — APPROVED** [review]
🥑

File: terraform/stacks/main/outputs.tf (hunk 1 above)
Comment (bengerman13) [reply terraform/stacks/main/outputs.tf:615]: I lied. It's an id

**Review by soutenniza — APPROVED** [review]
🐦

**Pull request merged**